### ISSUE TITLE
Rain cursors fill up the screen when using negative fallSpeed and skipIntro=false

### ISSUE THREAD

**Issue by hliang** [Rezmason/matrix#49]
The screen is gradually fill up the screen when using negative fallSpeed and skipIntro=false
example: https://rezmason.github.io/matrix/?effect=none&fallSpeed=-0.1&skipIntro=false

**Comment by antzpantz** [comment]
HAHA That's amazing! 😂

**Comment by Rezmason** [comment]
Yep, that's absolutely happening. 😅 If anyone has a suggestion for what the user should _expect_ when they do this, let me know, maybe I can make it happen.

**Comment by Rezmason** [comment]
Triaging tickets again, this really is the [none-pizza-with-left-beef](https://en.wikipedia.org/wiki/None_Pizza_with_Left_Beef) of this project 😆
